### ISSUE TITLE
Train Sepformer Model

### ISSUE THREAD

**Issue by dufaultc** [McMasterAI/isolate#18]
# Description
Need a Sepformer model trained on MUSDB dataset and the model file uploaded. Use the training steps in the sepformer_implementation notebook to get started.  Model should be trained until validation loss stops improving.

# Acceptance Criteria
- [ ] Model trained until validation loss stops improving.
- [ ] Models file uploaded in accessible location.
- [ ] Model training notebook added to repo with comments
